x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/ingest/SetSecurityUserProcessor.java

```diff
@@ -57,6 +62,7 @@ public SetSecurityUserProcessor(String tag, SecurityContext securityContext, XPa
         }
         this.field = field;
         this.properties = properties;
+        this.escCompliant = ecsCompliant;
```

Comment:
there are many places with the `esc` -> `ecs` inversion (including the PR title).

Reply:
Fixed most of them other than the one in the PR branch name. Thanks for spotting this.